src/wowman/core.clj

```diff
@@ -449,26 +515,17 @@
     (update-installed-addon-list! [])))
 
 ;;
-;; addon summaries
+;; catalog handling
 ;;
 
 (defn-spec download-catalog (s/or :ok ::sp/extant-file, :error nil?)
   "downloads catalog to expected location, nothing more"
   [& [catalog] (s/* keyword?)]
   (binding [http/*cache* (cache)]
     (if-let [local-catalog (paths (or catalog :catalog-file))]
-      (http/download-file remote-catalog local-catalog)
+      (http/download-file remote-catalog local-catalog :message "downloading catalog (~5MB)")
```

Comment:
double check that installed addons are displayed while the catalog is being downloaded. I suspect a regression here

Reply:
nope, looks fine